src/ops/sequences.jl

```diff
@@ -230,9 +247,9 @@ Returns:
 @op function Base.reverse(x::AbstractTensor, indices; name=nothing)
     local desc
     with_op_name(name, "Reverse") do
-        desc = NodeDescription("Reverse")
+        desc = NodeDescription("ReverseV2")
         add_input(desc, Tensor(x))
-        add_input(desc, Tensor(indices))
+        add_input(desc, cast(Tensor(indices), Int32)-1)
```

Comment:
Can this pattern `cast(Tensor(indices), Int32)-1)` be extracted out into a function.
It occurs in many places.